kqueen/storages/etcd.py

```diff
@@ -110,11 +109,11 @@ def validate(self):
         """
         return True
 
-    def _get_encryption_key(self):
-        """Read encryption key and format it.
+    def _get_fernet_obj(self):
```

Comment:
You may use AES instead of Fernet to be compatible with old database.

https://cryptography.io/en/latest/hazmat/primitives/symmetric-encryption/